tools/generate_tables_sql_and_testNames.php

```diff
@@ -2,19 +2,22 @@
 <?php
 
 /**
- * generate_tables_sql.php takes the ip_output.txt file generated from
+ * Generate_tables_sql.php takes the ip_output.txt file generated from
```

Comment:
that's nice, but what does this script (generate_tables_sql_and_testNames.php ) do?